src/modules/functions.js

```diff
@@ -0,0 +1,41 @@
+import taskList from './taskarray.js';
+import * as element from './elements.js';
+
+// Add Tasks to Array
+export const addTask = (x, y, z) => {
+  const obj = { a: x, b: y, c: z };
```

Comment:
- [ ] Rather than using letters, I recommend using meaningful names (ex: index, description, completed) so that others can understand your code.